### PR TITLE
Fixed dummy data generation

### PR DESCRIPTION
**Pull request by ayahusseini**
Now consistent with the new Schema, and doesn't attempt to insert IDs.

### CODE REVIEW

File: database/generate_dummy_data.py [hunk 1]
```diff
@@ -70,83 +63,81 @@ def generate_fake_subscribers(fake: Faker, num: int) -> list[tuple]:
...
-    
+
         for topic in all_topics:
-    
-            topic_id = TOPICS.index(topic) + 1 
-    
-            assignments.append((len(assignments) + 1, topic_id, article_id))
-    
+
+            topic_id = TOPICS.index(topic) + 1
```
Comment (megansam5) [inline database/generate_dummy_data.py:103]: Nice so not to override topic_id.

File: database/generate_dummy_data.py [hunk 2]
```diff
@@ -70,83 +63,81 @@ def generate_fake_subscribers(fake: Faker, num: int) -> list[tuple]:
...
-
         extras.execute_values(cursor, """
             INSERT INTO subscriber
-            (subscriber_id, subscriber_email, subscriber_first_name, 
+            (subscriber_email, subscriber_first_name, 
             subscriber_surname) 
             VALUES %s""", fake_subs)
 
-
-        extras.execute_values(cursor, """INSERT INTO article (article_id, article_title,
```
Comment (megansam5) [inline database/generate_dummy_data.py:133]: Good adjustment of removing article_id.

**Review by megansam5 — APPROVED** [review]

File: database/generate_dummy_data.py (hunk 1 above)
Comment (YakMan101) [inline database/generate_dummy_data.py:103]: Personal preference but I think we can remove the white lines between line 99 and 105 and the function still be clear to read.

File: database/generate_dummy_data.py [hunk 3]
```diff
@@ -70,83 +63,81 @@ def generate_fake_subscribers(fake: Faker, num: int) -> list[tuple]:
...
     return assignments
 
-def insert_data_to_db(conn, fake_topics: list[tuple], fake_subs: list[tuple], fake_articles: list[tuple], assignments:list[tuple]
-        ):
+
+def insert_data_to_db(conn, fake_subs: list[tuple], fake_articles: list[tuple], assignments: list[tuple]
+                      ):
     '''Inserts fake data into the database'''
 
     with conn.cursor() as cursor:
```
Comment (YakMan101) [inline database/generate_dummy_data.py:114]: good use of with for safe connection exiting

**Review by YakMan101 — APPROVED** [review]

**Pull request merged**